src/v4l2.c

```diff
@@ -544,6 +544,52 @@ print_v4l2_timeval(const MPERS_PTR_ARG(kernel_v4l2_timeval_t *) const arg)
 		print_v4l2_timeval(&((where_).field_));	\
 	} while (0)
 
+static void
+print_v4l2_buffer_contents(struct tcb *const tcp, const unsigned int code,
+		  const kernel_ulong_t arg, kernel_v4l2_buffer_t b)
```

Comment:
The `arg` argument seems to be unused here, let's drop it.
The `b` argument is a structure, let's pass a pointer instead.

Reply:
Addressed. Thanks!